src/test/ui/rfc-2632-const-trait-impl/feature-gate-const_trait_bound_opt_out.rs

```diff
@@ -0,0 +1,10 @@
+trait Trait {}
+struct Foo<T: Trait>(T);
+
+const fn new_foo<T: ?const Trait>(t: T) -> Foo<T> {
```

Comment:
I think the tests with and without the feature gate should have the same content so we are testing all the syntax in both situations

Reply:
I reorganized the tests a bit and am using the `revisions` trick for now to guarantee this.